gamemodes/terrortown/gamemode/shared/sh_player_ext.lua

```diff
@@ -101,6 +101,7 @@ if SERVER then
 	CreateConVar('ttt_armor_damage_health_pct', 0.7, {FCVAR_NOTIFY, FCVAR_ARCHIVE})
 	CreateConVar('ttt_armor_on_spawn', 0, {FCVAR_NOTIFY, FCVAR_ARCHIVE})
 	CreateConVar('ttt_armor_for_reinforced', 50, {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+	CreateConVar('ttt_armor_buy_value', 30, {FCVAR_NOTIFY, FCVAR_ARCHIVE})
```

Comment:
IMO the conVar should be named something like 'ttt_item_armor_value'.
For me personally 'ttt_armor_...' means it is a conVar directly related to the armorsystem. Throwing an item convar into it seems not a good idea, right?